pkg/routeagent/controllers/route/route.go

```diff
@@ -475,7 +490,7 @@ func (r *Controller) processNextEndpoint() bool {
 		}
 
 		r.endpointWorkqueue.Forget(obj)
-		klog.V(4).Infof("endpoint processed by route controller")
+		klog.V(log.DEBUG).Infof("endpoint processed by route controller")
```

Comment:
```suggestion
		klog.Infof("Successfully processed local Endpoint %q", endpoint.Name)
```

Reply:
But better as debug. Again, the initial purpose of this patch was to unclutter the logs, we are certainly making them better, but IMO that event belongs more into debug from the point of view of un-cluttering